docker-compose/goloop2moonbeam/Makefile

```diff
@@ -13,4 +13,12 @@ transfer_icx:
 	docker-compose exec btp_icon sh /btpsimple/scripts/transfer_icx.sh
 
 transfer_dev:
-	docker-compose exec btp_icon sh /btpsimple/scripts/transfer_dev.sh
\ No newline at end of file
+	docker-compose exec btp_icon sh /btpsimple/scripts/transfer_dev.sh
+
+# To run with gochain, please build gochain image from goloop source. 
+# Command line to build gochain: GL_TAG=0.9.7 make gochain-icon-image
```

Comment:
Why did env is GL_TAG=0.9.7, but in the  file of docker-compose-gochain.yml the variable GOLOOP_IMAGE: iconloop/goloop-icon:v0.9.9.